package.json

```diff
@@ -76,8 +76,6 @@
     "react-dom": "^15.4.2",
     "react-js-to-ts": "^1.4.0",
     "react-native": "0.61.4",
-    "react-native-keyboard-aware-scrollview": "^2.0.0",
-    "react-native-keyboard-tracking-view": "^5.6.1",
```

Comment:
let's try to make the library work while `react-native-keyboard-tracking-view` is installed because that's how it's going to be with the engine. 
as discussed, try to rename the native class names, it might fix the duplicate symbol issue that fail the build.

Reply:
This will be fixed in one of the other PRs